### PR TITLE
[update] Added double serialization for mappers before insert to db

### PR DESCRIPTION
**Pull request by romaBabyak**

### CODE REVIEW

File: Mappers/DbMapper.php [hunk 1]
```diff
@@ -105,7 +105,7 @@ public function load(SearchCriteria $filter) : PaginationCollection
      */
     public function create(array $objectData)
     {
-        return $this->createObject($objectData);
+        return $this->createObject($this->buildObject($objectData)->toArray());
```
Comment (Antonyan) [inline Mappers/DbMapper.php]: You should get ArraySerializible into createObject method, and do to array there

File: Mappers/DbMapper.php [hunk 2]
```diff
@@ -115,7 +115,7 @@ public function create(array $objectData)
      */
     public function update(array $objectData)
     {
-        return $this->updateObject($objectData);
+        return $this->updateObject($this->buildObject($objectData)->toArray());
```
Comment (Antonyan) [inline Mappers/DbMapper.php]: the same as above

File: Mappers/DbMapper.php (hunk 1 above)
Comment (romaBabyak) [reply Mappers/DbMapper.php]: did, please check

File: Mappers/DbMapper.php (hunk 2 above)
Comment (romaBabyak) [reply Mappers/DbMapper.php]: did, please check

**Review by Antonyan — APPROVED** [review]

**Pull request merged**